addons/website_event/views/event_templates_widgets.xml

```diff
@@ -2,6 +2,24 @@
 <odoo>
 
 <!-- Search Box -->
+<template id="events_search_box_input" name="Events search box">
+    <t t-call="website.website_search_box_input">
+        <t t-set="_classes" t-value="'o_wevent_event_searchbar_form w-100 my-1 my-lg-0 %s' % _classes"/>
+        <t t-set="search_type" t-valuef="events"/>
+        <t t-set="action" t-valuef="/event"/>
+        <t t-set="display_description" t-valuef="true"/>
+        <t t-set="display_detail" t-valuef="false"/>
```

Comment:
Are the *strings* `'true'` and `'false'` really what's expected / desired here?

Reply:
Yes, they actually are expected to be strings.